master/getting-started/docker/tutorials/basic.md

```diff
@@ -72,13 +71,7 @@ You should see output like this on each node
 This worked example creates three Docker networks, where containers on a
 particular network are isolated from containers in the other networks.
 
-### 3.1 Select the IPAM driver
-
-Before we create the networks, we need to select which IP address management
-(IPAM) driver we will use.  There are two options, where we suggest using the
-Calico IPAM driver where possible.
-
-#### The Calico IPAM driver
+### 3.1 The Calico IPAM driver
```

Comment:
Not sure we need such a prominent section on the IPAM driver now.   Basically you have to use it!  We can briefly describe what our IPAM driver does in terms of IP aggregation if we feel it necessary.